testutils/utils.go

```diff
@@ -10,19 +10,25 @@ import (
 
 // RunTestStore is an helper testing method that is
 // called by each K/V backend sub-package testing
-func RunTestStore(t *testing.T, kv store.Store, backup store.Store) {
+func RunTestStore(t *testing.T, kv store.Store) {
```

Comment:
What about splitting the tests by capabilities like this :

```
RunTestCommon:
  - testPutGetDeleteExists
  - testList
  - testDeleteTree

RunTestAtomic:
  - testAtomicPut
  - testAtomicPutCreate
  - testAtomicDelete

RunTestWatch:
  - testWatch
  - testWatchTree

RunTestLock:
  - testLockUnlock

RunTestTTL:
  - testPutTTL
```

Each set will probably see additional tests in the future. If we see that this is not enough with new Backends, we'll make all the methods public and each store will just call the supported subset.